test_onnx.py

```
@@ -0,0 +1,186 @@
+import pandas as pd
+from unittest.mock import MagicMock, patch
+from pathlib import Path
+import sys
+
+
+project_root = Path(__file__).parent
+sys.path.insert(0, str(project_root))
+
+from validator.validation_runner import ValidationRunner
+from validator.modules.onnx import (
+    ONNXValidationModule,
```

Ruff (0.12.2)

`validator.modules.onnx.ONNXValidationModule` imported but unused

Remove unused import

(F401)